src/client/components/collection/Item.js

```diff
@@ -89,7 +88,7 @@ export const Item = ({ item }) => {
       </Collapse>
 
       <CardActions disableSpacing>
-        {type === ITEM_TYPES.FOLDER && <CopyButton id={id} />}
+        {<CopyButton id={id} />}
```

Comment:
You don't need `{ }` around components, but only if you add a condition or use a function/variable.